src/server/CookingApp/ViewModels/Feedback/CreateFeedback.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace CookingApp.ViewModels.Feedback
+{
+    public class CreateFeedback
+    {
+        [MaxLength(50)]
```

Comment:
Create a constant class with the values as constants and use them instead of naked numbers.

Reply:
Perfect!